ignite/metrics/fbeta.py

```diff
@@ -56,8 +56,8 @@ def Fbeta(
 
         .. testcode:: 1
 
-            P = Precision(average=False)
-            R = Recall(average=False)
+            P = Precision(average=False)[1] # `[1]` is to select the value for class 1 that is TP/(TP+FP)
+            R = Recall(average=False)[1] # `[1]` is to select the value for class 1 that is TP/(TP+FN)
```

Comment:
@sadra-barikbin What we changed exactly in this PR (please provide the line of code) such that we have to update this example. I'm trying to understand if we are doing BC-breaking update or not

Reply:
https://github.com/sadra-barikbin/ignite/blob/214088b94c21471e4e287cefc411bc2f45bab3bb/ignite/metrics/precision.py#L311-L312

Reply:
Alternatively I can do this:
```python
P = Precision(average=False)
R = Recall(average=False)
metric = Fbeta(beta=1.0, precision=P, recall=R, average=False)[1]
```

Reply:
This change is because beforehand, Precision or recall with `average=False` returned a single float on binary input, but now they return an array with length 2.